ext/-test-/random/loop.c

```diff
@@ -0,0 +1,110 @@
+#include "ruby/random.h"
+
+static const uint32_t max_seeds = 1024;
+
+typedef struct {
+    rb_random_t base;
+    uint32_t num, idx, *buf;
+} rand_loop_t;
+
+static double loop_get_real(rb_random_t *rnd, int excl);
```

Comment:
@nobu How about adding `RB_RANDOM_INTERFACE_DECLARE_WITH_REAL` to declare functions with `get_real`?